WordPress/gutenberg
# [Mobile] - Slash inserter - Don't show disabled items

Fixes https://github.com/wordpress-mobile/gutenberg-mobile/issues/3584

`Gutenberg Mobile PR` -> https://github.com/wordpress-mobile/gutenberg-mobile/pull/3587

## Description
This PR filters out disabled items when using the Slash inserter.

## How has this been tested?
- Open the app with metro running
- Using either the slash inserter or the blocks menu, add the `Read more` block
- Try to add another `Read more` block **using the Slash inserter**
- **Expect** not to see the block in the autocomplete results since it was already added

## Screenshots 
Before|After
-|-
<kbd><img src="https://user-images.githubusercontent.com/4885740/120774736-dcdd9080-c522-11eb-92d4-8bda1912f604.gif" width="250" /></kbd> | <kbd><img src="https://user-images.githubusercontent.com/4885740/120774750-dfd88100-c522-11eb-9fa1-466241d8ca2b.gif" width="250" /></kbd>

## Types of changes
Bug fix

## Checklist:
